src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        int rand = random.nextInt(5);
+
+        switch (rand) {
+            case 0:
+                return new Circle(10, colorSupplier.getRandomColor());
+            case 1:
+                return new IsoscelesTrapezoid(10, 5, 2, colorSupplier.getRandomColor());
```

Comment:
Not necessary to write return in all cases, you can create variable on method level and populate it it cases